infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServer.java

```diff
@@ -63,33 +92,22 @@
     private final ConcurrentHashMap<UUID, Long> streamTailToGlobalTailMap = new ConcurrentHashMap<>();
 
     /**
-     * A cache of conflict parameter modification timestamps.
+     * A cache that records recent (stream, conflict parameter) modification
+     * timestamps (global addresses).
      *
-     * A "wildcard" {@link SequencerServer::maxConflictWildcard} representing all the entries which were evicted from the cache holds
-     * theier maximal modification time.
+     * And a "wildcard" representing all the entries which were evicted from the cache
+     * holds their maximal modification time.
      */
     private final long maxConflictCacheSize = 10_000;
     private long maxConflictWildcard = -1L;
-    private final Cache<Integer, Long> conflictToGlobalTailCache = Caffeine.newBuilder()
+    private final Cache<AbstractMap.SimpleEntry<Integer,Integer>, Long>
```

Comment:
Why is this a map entry now? Also if it's stream,conflictparameter why is it integer,integer?